### PR TITLE
Anton/add index for remove

### PR DESCRIPTION
**Pull request by antonoberg**

### CODE REVIEW

File: packages/backend/src/services/specimenService/resources/specimen/data/migrations/addIndex.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const relationshipIndexes = [
+  {
+    name: 'idx_spec_rel_phys',
```
Comment (fredrikolovsson) [inline packages/backend/src/services/specimenService/resources/specimen/data/migrations/addIndex.js]: not a fan of arbitrary names: phys, agent, place, taxa are 4 or 5 chars, singular, plural, and abbreviation all mixed.

how about something like this:
```javascript
const relationshipKeys = [
  'normalizedAgents',
  'physicalObjects',
  'places',
  'taxa',
]

const relationshipIndexes = relationshipKeys.map(relationship => {
  return {
    name: `idx_specimen_relationship_${relationship}`,
    relationship,
  }
})
```

File: packages/backend/src/lib/data/filters/factories/createManyJsonRelationshipFilter.js [hunk 2]
```diff
@@ -12,29 +12,30 @@ module.exports = function createManyJsonRelationshipFilter({
...
-                },
-              },
-            }
-          }),
-        },
+        [Op.or]: value.map(id => {
+          const regex = RegExp('^[0-9]+$')
+
+          if (!regex.test(id)) {
+            throw new Error(`id: ${id} is not allowed in filter`)
```
Comment (fredrikolovsson) [inline packages/backend/src/lib/data/filters/factories/createManyJsonRelationshipFilter.js]: nice! is not no longer sensitive for injections then?

File: packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/item/ActionBars/higherOrderComponents/createHandleDelete/InspectRelationsModal/RelationshipList.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      type: PropTypes.string.isRequired,
+    }),
+  ]),
+  fetchItemById: PropTypes.func.isRequired,
+}
+const defaultProps = {
+  data: {},
+}
+
+class ModalContent extends PureComponent {
```
Comment (fredrikolovsson) [inline packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/item/ActionBars/higherOrderComponents/createHandleDelete/InspectRelationsModal/RelationshipList.js]: fix name => RelationshipList

File: packages/backend/src/services/specimenService/resources/specimen/data/migrations/addIndex.js (hunk 1 above)
Comment (antonoberg) [reply packages/backend/src/services/specimenService/resources/specimen/data/migrations/addIndex.js]: #167cf3f

File: packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/item/ActionBars/higherOrderComponents/createHandleDelete/InspectRelationsModal/RelationshipList.js (hunk 3 above)
Comment (antonoberg) [reply packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/item/ActionBars/higherOrderComponents/createHandleDelete/InspectRelationsModal/RelationshipList.js]: f80e74a

**Pull request merged**